gnmi_cli_py/py_gnmicli.py

```diff
@@ -157,6 +157,8 @@ def _create_parser():
   parser.add_argument('--encoding', default=0, type=int, help='[0=JSON, 1=BYTES, 2=PROTO, 3=ASCII, 4=JSON_IETF]')
   parser.add_argument('--qos', default=0, type=int, help='')
   parser.add_argument('--use_alias', action='store_true', help='use alias')
+  parser.add_argument('--create_infinite_connections', action='store_true', help='trigger memory spike on gNMI server side'
```

Comment:
why always infinite? do we give options for user to specify the maximum connection they want to establish?

Reply:
for example, we can just setup 1000 connection to trigger memory spike. do we always run test with infinite connections, and always cause client to generate exceptions?

Reply:
> for example, we can just setup 1000 connection to trigger memory spike. do we always run test with infinite connections, and always cause client to generate exceptions?

The purpose of this option is to simulate the behavior of GWS C# gNMI client to trigger memory spike by creating TCP connections continuously as the following steps:

    1. telemetry server received initial query subscribe from Client_IP:port1
    2. Invalid db table Path STATE_DB DOCKER_STATS_TABLE,TEST_STATS_TABLE
    3. Client_IP:port1 shutdown
    4. telemetry server received initial query subscribe from Client_IP:port2
    5. Invalid db table Path STATE_DB DOCKER_STATS_TABLE,TEST_STATS_TABLE
    6. Client_IP:port2 shutdown
    7. Repeat from Step 1

In sonic-mgmt testing script, the gNMI client will be invoked with this option to create connections continuously with telemetry server side. Monit script will restart telemetry container if it catches the memory spike issue of telemetry. If telemetry server is restarted, gNMI clinet side will receive an notification/exception indicating telemetry server was shut down and then gNMI client side will exit directly.

Reply:
> why always infinite? do we give options for user to specify the maximum connection they want to establish?

I think we can give user an option to specify the maximum connections.